### PR TITLE
Finishing Analysis Work

### PR DESCRIPTION
**Pull request by corian-jal**
Tasks to Complete:
- Add functionality for pushing to the Google Drive
- Update all implementations to use the Google Drive and not local files
- Update the README with the data_analysis folder

### CODE REVIEW

**Comment by michplunkett** [comment]
I'm gonna get home tonight and I can take a look at it. After we merge this, if you could just add a sample of using it in the final Jupyter notebook you'd be all done!

**Comment by michplunkett** [comment]
And fantastic work! ☺️

File: data_analysis/night_record.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def __init__(
                 continue
             self.min_records.append(m_rec)
 
-        m_lines = docs.get_file(morning_file)
+        docs.get_file(morning_file)
         for line in n_lines:
```
Comment (michplunkett) [inline data_analysis/night_record.py]: Right here it looks like you aren't using the output from the morning file at all and are just running through the night file lines again.

**Review by michplunkett — CHANGES_REQUESTED** [review]

File: data_analysis/night_record.py (hunk 1 above)
Comment (corian-jal) [reply data_analysis/night_record.py]: Fixed!

**Review by michplunkett — APPROVED** [review]

**Pull request merged**